QuickSort.c

```diff
@@ -35,14 +38,14 @@ int main()
     printf("Enter the lenght of array: ");
     scanf("%d", &n);
     int arr[n];
-    for(i = 0; i < n; i++)
+    for (i = 0; i < n; i++)
     {
         printf("\nEnter the value of %d. element: ", i);
         scanf("%d", &arr[i]);
     }
 
     quickSort(arr, 0, n-1);

```

Comment:
Can you make everything before this line in `main` a separate function?

Reply:
I didn't use linter for the look.

Reply:
Do I need to initialize the i and n variables in that function or I can i main?

Reply:
How does it seems now?